episodes/continuous-integration.md

```diff
@@ -470,3 +480,11 @@ A small graph is shown and at the bottom "release-dists" link is provided.
 Click on that to download the package.
 
 That's it! Package published!
+
+## Considerations
+
+CI/CD pipelines are not very suitable if your tests requires a lot of static data.
+Running large integration tests inside a CI/CD pipeline is thus not recommended as there is generally limited space and time in CI/CD pipelines.
+Writing small and fast unit tests that run automatically inside CI/CD pipelines rather than large integration tests is recommended.
+It is therefore helpful to practice sotware engineering best practices such as decoupling.
+Decoupling enables testable code.
```

Comment:
```suggestion
It is therefore helpful to practice software engineering best practices such as decoupling, since that will lead to more easily testable code.
```